ui-components/src/forms/FieldGroup.tsx

```diff
@@ -1,14 +1,20 @@
-import React from "react"
+import React, { useState, useEffect } from "react"
```

Comment:
I think this component is a great candidate for unit testing components. With these updates, this is now probably one of the more complex field components we have, so it'd be great to get coverage on it with all its iterations.

Reply:
Added more tests